terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1,17 +1,18 @@
 environment = "staging"
 aws_profile = "staging-eu-west-2"
 
-api_url = "https://private-api-staging.company-information.service.gov.uk"
-cdn_host = "https://d6nh3dxv55e16.cloudfront.net"
-chs_url = "https://staging.company-information.service.gov.uk"
-cic_fee = 15
-cookie_domain = "company-information.service.gov.uk"
-cookie_name = "__SID"
+use_set_environment_files = true
 desired_task_count = 2 # use multi instance in live
-feedback_link = "https://www.smartsurvey.co.uk/s/file-package-accts-fdbk/"
 log_level = "info"
 node_env = "production"
-overseas_fee = 33
 piwik_url = "https://matomo.platform.aws.chdev.org"
-submit_validation_url = "https://test-validator-staging.companieshouse.gov.uk/xbrl_validate/submit"
 tz = "Europe/London"
+
+# Eric variables
+account_api_url = "https://identity-staging.company-information.service.gov.uk"
+developer_hub_url = "https://developer-staging.company-information.service.gov.uk"
```

Comment:
node_env, piwik_ur, tz, desired_task_count and the erik vars should also be removed and provided by the env file